app/src/main/java/sopt/org/millie/presentation/search/component/book/SearchWandokRate.kt

```diff
@@ -0,0 +1,65 @@
+package sopt.org.millie.presentation.search.component.book
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import sopt.org.millie.R
+import sopt.org.millie.core.designsystem.theme.MillieTheme
+
+@Composable
+fun SearchWandokRate(
+    rate: String,
+    minute: String,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        modifier = modifier,
+        horizontalArrangement = Arrangement.spacedBy(2.dp),
+        verticalAlignment = Alignment.CenterVertically,
+    ) {
+        Icon(
+            imageVector = ImageVector.vectorResource(id = R.drawable.ic_result_wandok),
+            contentDescription = null,
+            modifier = Modifier.size(15.dp),
+            tint = MillieTheme.colors.gray3,
+        )
+
+        Spacer(modifier = Modifier.height(2.dp))
```

Comment:
Spacer는 부모 레이아웃의 방향을 따라서 height라고 줘도 문제 없지만, Row 안에서는 보통 width로 가로 간격을 조정 해요 !
```suggestion
        Spacer(modifier = Modifier.width(2.dp))
```

Reply:
아 spacer은 height만 사용하는 줄 알았네용..ㅎㅎ 수정하겠습니다!